tests/Unit/Domain/Python/Test_ElementId.py

```diff
@@ -0,0 +1,37 @@
+# Distributed under the MIT License.
+# See LICENSE.txt for details.
+
+from spectre.Domain import ElementId1D, ElementId2D, ElementId3D, SegmentId
+import unittest
+
+
```

Comment:
You've created a binding for `external_boundary_id` but haven't tested it as far as I can tell.